test/transactions/helpers/MorphoAssociationTxTest.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./TxTest.sol";
+
+/// @notice The DAO can call all the governance functions including the ones that can be used by Morpho Operator.
+/// It just needs to be executed through the Delay Modifier.
```

Comment:
This is true if I remember well, you must go through the delay for some specific functions like market creation, upgrade, etc. but not everything. I need to double check

Reply:
Let's remove it then